src/main/java/com/quirko/gui/GuiController.java

```diff
@@ -147,12 +149,26 @@ public void initGameView(int[][] boardMatrix, ViewData brick) {
         brickPanel.setLayoutY(-42 + gamePanel.getLayoutY() + brick.getyPosition() * brickPanel.getHgap() + brick.getyPosition() * BRICK_SIZE);
 
         generatePreviewPanel(brick.getNextBrickData());
-
-
-        timeLine = new Timeline(new KeyFrame(
+        System.out.println(diff+" if else ");
+        if(diff.equalsIgnoreCase("easy")){
+            timeLine = new Timeline(new KeyFrame(
+                Duration.millis(1000),

```

Comment:
Instead of a fixed speed up approach, please think of another design. May be speed up over time little by little but use a different rate increment for each difficulty level.